app/src/main/java/com/codesoom/assignment/handler/TaskHttpHandler.java

```diff
@@ -0,0 +1,119 @@
+package com.codesoom.assignment.handler;
+
+import com.codesoom.assignment.http.HttpStatus;
+import com.codesoom.assignment.http.RequestMethod;
+import com.codesoom.assignment.http.RequestUtils;
+import com.codesoom.assignment.http.ResponseUtils;
+import com.codesoom.assignment.model.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class TaskHttpHandler implements HttpHandler {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private final List<Task> tasks = new ArrayList<>();
+
+    private final Long ID_NOT_GIVEN = -1L;
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        URI uri = exchange.getRequestURI();
+        String requestURI = uri.getPath();
+        String requestBody = RequestUtils.readRequestBody(exchange.getRequestBody());
+        RequestMethod requestMethod = RequestMethod.valueOf(exchange.getRequestMethod());
+
+        if (!RequestUtils.isValidRequest(requestMethod, requestURI, requestBody)) {
+            ResponseUtils.sendError(exchange, HttpStatus.BAD_REQUEST);
+        }
+
+        Optional<Long> optionalId = RequestUtils.getResourceId(requestURI);
+        Long id = optionalId.orElse(ID_NOT_GIVEN);
```

Comment:
```suggestion
        Long id = RequestUtils.getResourceId(requestURI)
                                               .optionalId.orElse(ID_NOT_GIVEN);
```

`optionalId`를 다른 곳에서 사용하지 않는다면 사용할 `id` 만 변수로 지정하는 것은 어떨까요?!

Reply:
optionalId 변수가 따로 만들어질 필요는 없겠네요! 수정하겠습니다